src/main/java/com/jayway/maven/plugins/android/phase_prebuild/ClasspathModifierLifecycleParticipant.java

```diff
@@ -285,5 +292,25 @@ private Dependency createSystemScopeDependency( Artifact artifact, File location
         dependency.setSystemPath( location.getAbsolutePath() );
         return dependency;
     }
+    
+    private Dependency findProvidedDependencies( Dependency dexDependency, MavenProject project ) 
+    {
+        for ( Dependency dependency : project.getDependencies() ) 
+        {
+            if ( dependency.getScope().equals( Artifact.SCOPE_PROVIDED ) ) 
```

Comment:
Are they always going to be SCOPE_PROVIDED?

What dependency are you looking for the AAR or it's contained JAR?
NB dexDependency.getType() hasn't been defined when it was constructed so at best you are getting the default type.